### ISSUE TITLE
Match img redundant alt with word boundary `\b`

### ISSUE THREAD

**Issue by Moeriki** [evcohen/eslint-plugin-jsx-a11y#164]
I'm making a website for a photographer.

```jsx
<img src="" alt="Sandy Croes Photography" />
```

The above jsx triggers `img-redundant-alt` for obvious reasons.

As it is, the regex is matched without word boundary `\b`.

```js
const hasRedundancy = redundantWords.some(
  word => Boolean(value.match(new RegExp(`(?!{)${word}(?!})`, 'gi')))
);
```

Would you agree it is a good idea to update the regex to:

```js
const hasRedundancy = redundantWords.some(
  word => Boolean(value.match(new RegExp(`\\b(?!{)${word}(?!})\\b`, 'i')))
);
```

As a side-note I think the `g` flag is superfluous.

**Comment by evcohen** [comment]
agreed! this LGTM, good find. mind sending a PR?
